src/frontend/src/pages/GanttPage/GanttChartComponents/GanttTaskBar/GanttTaskBarDisplayStyles.tsx

```diff
@@ -0,0 +1,54 @@
+import { CSSProperties } from 'react';
+import { GanttTask, GANTT_CHART_CELL_SIZE, GANTT_CHART_GAP_SIZE } from '../../../../utils/gantt.utils';
+
+export const ganttTaskBarBackgroundStyles = (numDays: number): CSSProperties => {
+  return {
+    width: '100%',
+    display: 'grid',
+    gap: GANTT_CHART_GAP_SIZE,
+    gridTemplateRows: `repeat(1, minmax(0, 1fr))`,
+    gridTemplateColumns: `repeat(${numDays}, minmax(${GANTT_CHART_CELL_SIZE}, 1fr))`
+    //  position: 'absolute', // These will make it so that the bar stays on top of the drop points, i kind of like it going to a new line though
```

Comment:
if you're talking about when it reorders so that the earliest is on top, and drops the changed element to a new line, I like that too. You can delete these comments if you like how it works without them

Reply:
no im talking about when you change start date the line with like the cells appears and moves the actual work package to a line below it